src/ceph-disk/ceph_disk/main.py

```diff
@@ -4743,6 +4758,12 @@ def make_activate_parser(subparsers):
         action='store_true', default=False,
         help='activate the deactived OSD',
     )
+    activate_parser.add_argument(
+        '--cluster',
+        metavar='NAME',
+        default='ceph',
+        help='ceph cluster name (default: ceph)',
+    )
```

Comment:
Alright, let me try again without this, I forgot the error that made us think we needed to do this.